apps/pie-docs/src/assets/js/search.js

```diff
@@ -0,0 +1,80 @@
+window.addEventListener('DOMContentLoaded', () => {
+    // eslint-disable-next-line
+    docsearch({
+        // public algolia keys (can be exposed)
+        appId: 'L21G25CLL1',
+        apiKey: '75fcb9c4b333a7a004ae63e6a736a81e',
+        indexName: 'pie',
+        container: '#docsearch',
+    });
+
+    document.querySelector('.DocSearch').onclick = () => {
+        const startScreenOptions = [
+            {
+                category: {
```

Comment:
Nothing that needs changing now, but there is a bit of duplication here and usage of magic strings. If a page title is altered, we need to update here too. Same for URLs. 

It might be that we can generate this object instead. It doesn't need to be done as a part of this PR, but it could be a useful task later on 😃

Reply:
agreed, will resolve in the next PR when I add some tests 🚀 thank you!